usmqe_tests/api/grafana/conftest.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import os.path
+import datetime
+import time
+import usmqe.usmssh
+from usmqe.api.tendrlapi import glusterapi
+from usmqe.gluster import gluster
+
+LOGGER = pytest.get_logger('grafana_conftest', module=True)
+
+
+def measure_operation(operation):
+    """
+    Get dictionary with keys 'start', 'end' and 'result' that contain
+    information about start and stop time of given function and its result.
+
+    Args:
+        operation (function): Function to be performed.
+
+    Returns:
+        dict: contains information about `start` and `stop` time of given
+            function and its `result`
+    """
+    start_time = datetime.datetime.now()
+    result = operation()
+    end_time = datetime.datetime.now()
+    return {
+        "start": start_time,
+        "end": end_time,
+        "result": result}
+
+
+@pytest.fixture(params=[60, 80, 95])
+def measured_cpu_utilization(request):
+    """
+    Returns:
+        dict: contains information about `start` and `stop` time of stress-ng
+            command and its `result`
+    """
+    def fill_cpu():
+        """
+        Use `stress-ng` tool to stress cpu for 5 seconds to given percentage
+        """
+        # stress cpu for for 5 seconds
+        run_time = 5
+        SSH = usmqe.usmssh.get_ssh()
+        host = pytest.config.getini("usm_cluster_member")
+        processors_cmd = "grep -c ^processor /proc/cpuinfo"
+        retcode, processors_count, _ = SSH[host].run(processors_cmd)
+        stress_cmd = "stress-ng --cpu {} -l {} --timeout {}s".format(
```

Comment:
Installation of stress-ng (from epel) could be handled by ansible playbook or role. So far, we don't need anything special to be installed on storage machines.

Reply:
Update: installation of stress-ng is handled by https://github.com/usmqe/usmqe-setup/blob/master/test_setup.stress_ng.yml. There is also a teardown playbook.